components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -3164,14 +3167,24 @@ public static boolean isJITProvisionEnhancedFeatureEnabled() {
      * @throws ClaimManagementException
      */
     public static List<ClaimMapping> getFilteredScopeClaims(List<String> claimListOfScopes,
-                                                            List<ClaimMapping> claimMappings)
-            throws ClaimManagementException {
+                                                            List<ClaimMapping> claimMappings, String tenantDomain)
+            throws ClaimManagementException, CarbonException, UserStoreException {
```

Comment:
let's catch these exceptions inside the code and throw as ClaimManagementException

Reply:
Addressed with commit 6f87afcaee2129b1d3f3c7f8d82209f9d5a982cf